platform/linux-generic/arch/aarch64/odp_crypto_armv8.c

```diff
@@ -131,6 +152,212 @@ null_crypto_routine(odp_packet_t pkt ODP_UNUSED,
 	return ODP_CRYPTO_ALG_ERR_NONE;
 }
 
+static
+odp_crypto_alg_err_t aes_gcm_encrypt(odp_packet_t pkt,
+				     const odp_crypto_packet_op_param_t *param,
+				     odp_crypto_generic_session_t *session)
+{
+	armv8_cipher_state_t cs = {
+		.counter = {
+			.d = {0, 0}
+		}
+	};
+	uint8_t *iv_ptr;
+	uint64_t plaintext_bit_length = (param->cipher_range.length) * 8;
+	uint64_t iv_bit_length = (session->p.cipher_iv.length) * 8;
+	uint64_t aad_bit_length = (session->p.auth_aad_len) * 8;
```

Comment:
The unnecessary parentheses could be removed.

Reply:
I'll make the change. Thanks